packages/react-composites/tests/browser/chat/fake-adapter/ErrorBar.test.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { stubMessageTimestamps, waitForChatCompositeToLoad, dataUiId, pageClick } from '../../common/utils';
+import { buildUrlForChatAppUsingFakeAdapter, DEFAULT_FAKE_CHAT_ADAPTER_ARGS, test } from './fixture';
+import { expect } from '@playwright/test';
+import { sendMessage, waitForSendMessageFailure } from '../../common/chatTestHelpers';
+
+const TEST_MESSAGE = 'No, sir, this will not do.';
+
+test.describe('ErrorBar is shown correctly', async () => {
+  test('with wrong thread ID', async ({ page, serverUrl }) => {
+    await page.goto(
+      buildUrlForChatAppUsingFakeAdapter(serverUrl, {
+        ...DEFAULT_FAKE_CHAT_ADAPTER_ARGS,
+        chatThreadClientFunctionErrors: {
+          getProperties: { message: 'Could not get properties', statusCode: 400 },
```

Comment:
Did you check these return codes by running the real test?

Reply:
Yes I did.